tests/test_engine/test_model_traffic_capture.py

```
@@ -127,17 +129,45 @@ def _jsonl(path: Path):
+                    "tool_calls": tool_calls
+                    or [{"id": "c1", "type": "function", "function": {"name": "w", "arguments": "{}"}}],
+                },
+                "finish_reason": "tool_calls",
+            }
+        ],
+        "usage": {"prompt_tokens": 3, "completion_tokens": 4, "total_tokens": 7},
+    }
+
+
+@pytest.mark.parametrize(
+    "capture_kwargs,expect_response_fields",
```

Ruff (0.15.18)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)